packages/vitest/package.json

```diff
@@ -2,9 +2,12 @@
   "name": "@effection/vitest",
   "version": "2.0.1",
   "description": "Effection Integration for the Vitest framework",
-  "main": "dist-cjs/index.js",
-  "module": "dist-esm/index.js",
   "types": "dist-esm/index.d.ts",
+  "type": "module",
```

Comment:
I think this should be safe as vitest is ESM first, yea? Any caveats here we may need to think about?

Reply:
vitest is esm only from what I can gather.  I don't think it make sense to have a commonjs build